librz/analysis/sign.c

```diff
@@ -1853,25 +1853,25 @@ static void listXRefs(RzAnalysis *a, RzSignItem *it, PJ *pj, int format) {
 	}
 }
 
-static void listRefs(RzAnalysis *a, RzSignItem *it, PJ *pj, int format) {
+static void listXRefsFrom(RzAnalysis *a, RzSignItem *it, PJ *pj, int format) {
 	RzListIter *iter = NULL;
 	char *ref = NULL;
 	int i = 0;
 
 	if (format == '*') {
 		a->cb_printf("za %s r ", it->name);
 	} else if (format == 'q') {
-		a->cb_printf(" refs(%d)", rz_list_length(it->refs));
+		a->cb_printf(" xrefs_from(%d)", rz_list_length(it->xrefs_from));
```

Comment:
I am not sure it should affect user-visible output though. Most people got used to the `refs` and `xrefs`.

Reply:
I think "xrefs from here" and "xrefs to here" is better even in user output. I personally never really got used to refs/xrefs.

Reply:
I agree with @thestr4ng3r